src/main/resources/db/migrationv2/V2_43__housekeeping_trial_deletion_update.sql

```diff
@@ -0,0 +1,77 @@
+-- DROP PROCEDURE juror_mod.housekeeping_trial_deletion(in int4, in int4, in int4, inout text);
+
+CREATE OR REPLACE PROCEDURE juror_mod.housekeeping_trial_deletion(IN p_max_threshold integer, IN p_start_time_int integer, IN p_max_timeout integer, INOUT p_print_msg text)
+ LANGUAGE plpgsql
+AS $procedure$
+DECLARE
+   	v_text_var1 TEXT;
+   	v_text_var2 TEXT;
+   	v_text_var3 TEXT;
+   	v_timed_out BOOLEAN;
+
+BEGIN
+	<<Deletes>>
+	BEGIN
+		-- child records
+		WITH aged_trials AS
+		(
+			SELECT t.trial_number
+			FROM juror_mod.trial t
+			WHERE t.trial_end_date < CURRENT_DATE - p_max_threshold
+		), excluded_trials AS
+    (
+      SELECT jt.trial_number
+      FROM juror_mod.juror_trial jt
+      INNER JOIN aged_trials agt ON jt.trial_number = agt.trial_number
+    )
+		DELETE
+		FROM juror_mod.accused a
+		USING aged_trials agt
+		WHERE a.trial_no = agt.trial_number and agt.trial_number not in (select et.trial_number from excluded_trials et);
```

Comment:
Need to join a with agt using loc_code as well trial_no.
Need to use et.loc_code as well trial_no to identify excluded trials.